src/spawner/Spawner.js

```diff
@@ -2,57 +2,68 @@ import math.geom.Point as Point;
 
 import scene.group.Group as Group;
 
+/**
+ * @callback SpawnFunction
+ * @arg    {number}  x
+ * @arg    {number}  y
+ * @arg    {number}  index Index (since beginning of game) of the currently spawned element
+ * @arg    {Spawner} spawner Spawner instance
+ * @this   {Spawner}
+ * @return {Actor|Actor[]} newly spawned actor/s
+ */
+
+/** @lends Spawner */
 exports = Class(function() {
+
   /**
-    * @callback SpawnFunction
-    * @arg {number} x
-    * @arg {number} y
-    * @arg {number} index - the index (since beginning of game) of the currently spawned element
-    * @arg {Spawner} spawner - the spawner instance
-    * @this {Spawner}
-    * @return {Actor|Actor[]} newly spawned actor/s
-    */
-  /**
-    * Calls {@link Spawner#spawnFunction} every {@link Spawner#spawnDelay}
-    * @class Spawner
-    * @arg {Shape} spawnAt
-    * @arg {SpawnFunction} spawnFunction
-    * @arg {Number} spawnDelay - Either expressed as pixels or as milliseconds until the next spawn
-    * @arg {Boolean} useWorldSpace (false) - positions will be considered in-world positions
-    */
+   * Calls {@link Spawner#spawnFunction} every {@link Spawner#spawnDelay}
+   * @constructs
+   * @arg {Shape}         spawnAt
+   * @arg {SpawnFunction} spawnFunction
+   * @arg {number}        [spawnDelay]     Either expressed as pixels or as milliseconds until the next spawn
+   * @arg {boolean}       [useWorldSpace]  Positions will be considered in-world positions
```

Comment:
Could better define what `WorldSpace` literally means. I'm guessing it means whether or not to account for the camera position?